tests/test_sessions.py

```diff
@@ -0,0 +1,51 @@
+from bocadillo import App, view
+from bocadillo.testing import create_client
+
+
+def test_sessions_enabled_no_secret_key():
+    app = App(enable_sessions=True)
+
+    @app.route("/")
+    async def index(req, res):
+        res.text = "Hello"
+
+    client = create_client(app)
+    response = client.get("/")
+    assert len(response.cookies) == 0
+
+
+def test_sessions_enabled_secret_key_empty(mocker):
+    mocker.patch.dict("os.environ", {"SECRET_KEY": ""})
```

Comment:
I believe the built-in `override_env` utility located in `bocadillo.utils` could prevent us from adding a dependency on `pytest-mock` here. It allows to temporarily override an environment variable.